### ISSUE TITLE
Nov 10-11: Build a mobile app with Xamarin

### ISSUE THREAD

**Issue by mebaumb** [microsoft/ReactorPlanning#115]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Coding, Languages, and Frameworks |
| Monthly Topic (if applicable) | .NET November |
| Workshop Duration | 60 min | 
| Title | Build a Mobile App with Xamarin |
| Description | Harness the power of Xamarin to help your project go mobile! In this workshop, we'll start off by learning the basic structure of Xamarin.Forms. Then we'll use Visual Studio to create a phone number translator app that runs on Android and iOS. No previous experience with Xamarin or C# is needed. Get your free Microsoft account and download Visual Studio to build the app yourself! We'll be covering this Learn module: https://aka.ms/XamarinFormsApp-2 if you'd like to follow along. |
| Content | https://aka.ms/XamarinFormsApp-8 |

# Workshop(s) details
| Date |  Local Start Time |  Location |  Speaker | SharePoint ID | Platform |
|------|-------------------|-----------|----------|---------------|----------|
| 11/10 | 6:30 PM IST | Bangalore | Devendra Rawat | 11870 |----------|
| 11/10 | 4:00 PM PST | US, Canada, Sydney | Allen Sanders | 11871 |----------|
| 11/11 | 7:30 PM | Sao Paulo |----------| 11872 |----------|
| 11/11 | 7:30 PM | Shanghai | Warren Wong | 11873 |----------|
| 11/11 | 12:00 PM BST | London, Stockholm, Abu Dhabi, Tel Aviv | Allen Sanders | 11874 |----------|

# Event Checklist

## 10 Weeks Prior
- [x] Reactor Content Team (RCT) opens a new issue for an upcoming local workshop filling in the Event Information and Workshop details tables. RCT member who owns content (content owner) this week will be the assignee.
- [x] RCT and content owner discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _local-digital-event_ tag to the issue and adds event to the __Local Digital Events__ project. Issue will appear in the __Upcoming Streams__ column
- [x] RCT adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [x] RCT informs the Reactor Marketing Team of the new event information
- [x] RCT confirms the speaker
- [x] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [x] RCT moves the issue to the __Content Preparation In Progress__ column

## 5 weeks Prior
- [ ] Content owner opens a PR with the new content for the workshop on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors/tree/main/online-event-resources) and adds the _content-development_ and _local-digital-event_ tag to the PR. 
- [ ] RCT reviews, approves, and merges the changes
- [ ] RCT shares content with the vendor/instructor
- [ ] RCT provides vendor/instructor with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT ensures that instructors have received an invite to the workshops and that PM team works with instructors to get setup in the platform (e.g. Teams, Teams Live Event)

## 1 Week Prior
- [ ] Connect vendor/instructor with the PM(s) for the locations where this event has been marketed
- [ ] Check-in with vendor/instructor that they are prepared for the workshop
- [ ] RCT moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Instructor (and optional moderator) run the stream
- [ ] _(Optional)_RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by mebaumb** [comment]
Are times going to be the same as the previous week? 

Here is a pass on an edit: 

Harness the power of Xamarian to help your project go mobile! In this workshop, we'll start off by learning the basic structure and then work up to using Xamarian to develop a mobile app. Then we'll use Visual Studio to create a phone number translator app that runs on Android and iOS. 

No previous experience with Xamarin or C# is needed. Get your free Microsoft account to follow along in the workshop <link to module>!

**Comment by meaghanlewis** [comment]
Yes, times will be the same this week! I made slight updates to the description below:

Harness the power of Xamarian to help your project go mobile! In this workshop, we'll start off by learning the basic structure of Xamarin.Forms. Then we'll use Visual Studio to create a phone number translator app that runs on Android and iOS.

No previous experience with Xamarin or C# is needed. Get your free Microsoft account and download Visual Studio to follow along in the workshop!

**Comment by mebaumb** [comment]
Looks great to me! @meaghanlewis are we good to publish? or are we still waiting on confirmation for dates/times or anything else?

**Comment by meaghanlewis** [comment]
We are good to publish @mebaumb !

**Comment by mebaumb** [comment]
This has  been published, we are waiting for speakers, will add those in when I have them!

**Comment by mebaumb** [comment]
Speakers updated and published!

**Comment by meaghanlewis** [comment]
Here's an overview of what this workshop will cover and suggested agenda to follow with links and resources.

## Content to teach
This workshop will take Reactor community members through the Microsoft Learn module: [Create a mobile app with Xamarin.Forms](https://aka.ms/XamarinFormsApp-8)

**SHARE THIS LINK WITH REACTOR ATTENDEES: https://aka.ms/XamarinFormsApp-8**

**Encourage attendees to go through the modules with you and complete it. Also encourage them to rate the module at the end.**

## Agenda
1. [Intro](https://docs.microsoft.com/en-us/learn/modules/create-a-mobile-app-with-xamarin-forms/1-introduction) 3 min
   - Welcome community members
   - Provide overview of content to be covered
   - Explain what Xamarin is and what it's used for
   - Explain how they can follow along during the workshop
      - **Visual Studio** is needed
      - Familiarity with C# and .NET encouraged
   - Encourage community members to complete and rate the module
2. [Create a Xamarin forms project](https://docs.microsoft.com/en-us/learn/modules/create-a-mobile-app-with-xamarin-forms/2-create-a-xf-project-in-vs) 10 min
    - Highlight more information about Xamarin.Forms
    - Create a new project
    - Go over the knowledge check questions, giving attendees a moment to answer on their own
3. [Create your first Xamarin.Forms application](https://docs.microsoft.com/en-us/learn/modules/create-a-mobile-app-with-xamarin-forms/3-exercise-create-your-first-xamarin-forms-application) 5 min
    - Walk through the steps to complete a project and application
4. [Add visual controls to a Xamarin.Forms mobile app](https://docs.microsoft.com/en-us/learn/modules/create-a-mobile-app-with-xamarin-forms/4-design-your-first-xamarin-forms-ui) 10 min
    - Talk through the different components of a Xamarin.Forms project
    - Build the content page
    - Go over the knowledge check questions, giving attendees a moment to answer on their own
5. [Create a phone number translator app](https://docs.microsoft.com/en-us/learn/modules/create-a-mobile-app-with-xamarin-forms/5-exercise-create-phone-number-translator-app) 25 min
    - Walk through the steps to build the Phoneword app
6. **Next steps** 2 min
    - Encourage attendees to complete the summary and complete and rate the module
7. **Closing** 5 min
    - Q&A
    - **Ask attendees to take the survey: https://aka.ms/Reactor/Survey**
      - Note: The event code is listed in this issue description and/or your Reactor contact can share that with the community members during the workshop
    - Share Resources
      - Encourage attendees to visit: https://docs.microsoft.com/learn/browse/?terms=xamarin for other Xamarin content on Microsoft Learn
    - Check if the Reactor contact has other information to share

**Comment by mebaumb** [comment]
This event has been delivered and the attendance data recorded. Closing.
